import org.apache.hadoop.fs.shell.PathExceptions.PathIOException;
public class FunctionUse5902 {
public void funcUse() {
PathIOException pathioexception = new PathIOException();
pathioexception.formatPath(null);
}
}